src/components/containers/show.js

```diff
@@ -13,55 +12,35 @@ const handleEu = (setVisible, to) => is_eu_country => {
         case 'non-eu':
             setVisible(!is_eu_country)
             break
-        case 'global':
-            setVisible(true)
-            break
         default:
             break
     }
 }
 
-const Show = ({ children, to = 'global', device }) => {
+const Show = ({ children, to }) => {
     const [visible, setVisible] = useState(false)
-    const [deviceVisibility, setdeviceVisibility] = useState(true)
     useEffect(() => {
-        if (to) {
-            const clients_country = Cookies.get('clients_country')
-            const showEu = handleEu(setVisible, to)
-            if (clients_country) {
-                isEuCountry(clients_country)
-                showEu(isEuCountry(clients_country))
-            } else {
-                BinarySocketBase.wait('website_status').then(response => {
-                    showEu(isEuCountry(response.website_status.clients_country))
+        if (!to) return
 
-                    /* country_code cookies will be valid for 1 month */
-                    Cookies.set(
-                        'clients_country',
-                        response.website_status.clients_country,
-                        { expires: 30 },
-                    )
-                })
-            }
-        }
-        if (device) {
-            switch (device) {
-                case 'laptop':
-                    setdeviceVisibility(
-                        window.innerWidth > size.tabletL.slice(0, -2),
-                    )
-                    break
-                case 'mobile':
-                    setdeviceVisibility(
-                        window.innerWidth < size.tabletL.slice(0, -2),
-                    )
-                    break
-                default:
-                    break
-            }
+        const clients_country = Cookies.get('clients_country')
+        const showEu = handleEu(setVisible, to)
+        if (clients_country) {
+            isEuCountry(clients_country)
```

Comment:
```suggestion
          //  isEuCountry(clients_country)
```

Reply:
we don't use show component anywhere at all as of now. we probably use this only if there is small difference between eu and non eu in a single page. otherwise we will remove the show component.